lib/core/macros.nim

```diff
@@ -177,6 +177,12 @@ proc getType*(n: NimNode): NimNode {.magic: "NGetType", noSideEffect.}
   ## resolve recursive types, you have to call 'getType' again. To see what
   ## kind of type it is, call `typeKind` on getType's result.
 
+proc typeNode*(n: typedesc): NimNode {.magic: "NGetType", noSideEffect.}
```

Comment:
Okay, no problem, the reason I decided to call it `typeNode` was that `SomeType.getType` sounds just weird as "getting type of type !?" Another reason was `typeKind` proc (same `NGetType` magic), I think `typeNode` matches this better. But if you think `getType` won't be confusing then OK.